internal/server/providers/_testdata/test-server-sec.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This looks alarming but its only used by the test server for signing test ID tokens.